Created by Jasper Grimm (UoY)
Patch include statements to match installed include hierarchy
diff -Nru freebayes-1.3.5/src/Allele.cpp freebayes-1.3.5b/src/Allele.cpp
--- freebayes-1.3.5/src/Allele.cpp	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/Allele.cpp	2021-05-13 16:26:23.000000000 +0100
@@ -1,5 +1,5 @@
 #include "Allele.h"
-#include "multichoose.h"
+#include "multichoose/multichoose.h"
 #include "TryCatch.h"
 
 
diff -Nru freebayes-1.3.5/src/AlleleParser.cpp freebayes-1.3.5b/src/AlleleParser.cpp
--- freebayes-1.3.5/src/AlleleParser.cpp	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/AlleleParser.cpp	2021-05-13 16:26:24.000000000 +0100
@@ -1,9 +1,9 @@
 #include "AlleleParser.h"
-#include "multichoose.h" // includes generic functions, so it must be included here
+#include "multichoose/multichoose.h" // includes generic functions, so it must be included here
                          // otherwise we will get a linker error
                          // see: http://stackoverflow.com/questions/36039/templates-spread-across-multiple-files
                          // http://www.cplusplus.com/doc/tutorial/templates/ "Templates and Multi-file projects"
-#include "multipermute.h"
+#include "multichoose/multipermute.h"
 #include "Logging.h"
 
 using namespace std;
diff -Nru freebayes-1.3.5/src/AlleleParser.h freebayes-1.3.5b/src/AlleleParser.h
--- freebayes-1.3.5/src/AlleleParser.h	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/AlleleParser.h	2021-05-13 16:26:24.000000000 +0100
@@ -13,7 +13,7 @@
 #include <assert.h>
 #include <ctype.h>
 #include <cmath>
-#include "split.h"
+#include "fastahack/split.h"
 #include <list> // XXX workaround for a missing include in vcflib's join.h
 #include "join.h"
 
diff -Nru freebayes-1.3.5/src/BedReader.cpp freebayes-1.3.5b/src/BedReader.cpp
--- freebayes-1.3.5/src/BedReader.cpp	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/BedReader.cpp	2021-05-13 16:26:23.000000000 +0100
@@ -7,10 +7,10 @@
 #include <iterator>
 #include <algorithm>
 #include <cmath>
-#include "split.h"
+#include "fastahack/split.h"
 #include "Utility.h"
 #include "BedReader.h"
-#include "../intervaltree/IntervalTree.h"
+#include "intervaltree/IntervalTree.h"
 
 using namespace std;
 
diff -Nru freebayes-1.3.5/src/BedReader.h freebayes-1.3.5b/src/BedReader.h
--- freebayes-1.3.5/src/BedReader.h	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/BedReader.h	2021-05-13 16:26:23.000000000 +0100
@@ -9,8 +9,8 @@
 #include <map>
 #include <iterator>
 #include <algorithm>
-#include "../intervaltree/IntervalTree.h"
-#include "split.h"
+#include "intervaltree/IntervalTree.h"
+#include "fastahack/split.h"
 
 using namespace std;
 
diff -Nru freebayes-1.3.5/src/Bias.h freebayes-1.3.5b/src/Bias.h
--- freebayes-1.3.5/src/Bias.h	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/Bias.h	2021-05-13 16:26:24.000000000 +0100
@@ -7,7 +7,7 @@
 #include <string>
 #include <vector>
 #include <cstdlib>
-#include "split.h"
+#include "fastahack/split.h"
 
 using namespace std;
 
diff -Nru freebayes-1.3.5/src/CNV.h freebayes-1.3.5b/src/CNV.h
--- freebayes-1.3.5/src/CNV.h	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/CNV.h	2021-05-13 16:26:24.000000000 +0100
@@ -10,7 +10,7 @@
 #include <stdlib.h>
 #include <algorithm>
 #include <tuple>
-#include "split.h"
+#include "fastahack/split.h"
 
 using namespace std;
 
diff -Nru freebayes-1.3.5/src/Contamination.h freebayes-1.3.5b/src/Contamination.h
--- freebayes-1.3.5/src/Contamination.h	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/Contamination.h	2021-05-13 16:26:24.000000000 +0100
@@ -8,7 +8,7 @@
 #include <vector>
 #include <cstdlib>
 #include <cmath>
-#include "split.h"
+#include "fastahack/split.h"
 
 using namespace std;
 
diff -Nru freebayes-1.3.5/src/DataLikelihood.cpp freebayes-1.3.5b/src/DataLikelihood.cpp
--- freebayes-1.3.5/src/DataLikelihood.cpp	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/DataLikelihood.cpp	2021-05-13 16:26:24.000000000 +0100
@@ -1,6 +1,6 @@
 #include "DataLikelihood.h"
-#include "multichoose.h"
-#include "multipermute.h"
+#include "multichoose/multichoose.h"
+#include "multichoose/multipermute.h"
 #include "Logging.h"
 
 long double
diff -Nru freebayes-1.3.5/src/FBFasta.h freebayes-1.3.5b/src/FBFasta.h
--- freebayes-1.3.5/src/FBFasta.h	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/FBFasta.h	2021-05-13 16:26:24.000000000 +0100
@@ -16,10 +16,10 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <algorithm>
-#include "LargeFileSupport.h"
+#include "fastahack/LargeFileSupport.h"
 #include "Utility.h"
 #include <sys/stat.h>
-#include "split.h"
+#include "fastahack/split.h"
 #include <stdlib.h>
 #include <ctype.h>
 #include <unistd.h>
diff -Nru freebayes-1.3.5/src/freebayes.cpp freebayes-1.3.5b/src/freebayes.cpp
--- freebayes-1.3.5/src/freebayes.cpp	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/freebayes.cpp	2021-05-13 16:26:23.000000000 +0100
@@ -30,8 +30,8 @@
 #include "Utility.h"
 #include "SegfaultHandler.h"
 
-#include "multichoose.h"
-#include "multipermute.h"
+#include "multichoose/multichoose.h"
+#include "multichoose/multipermute.h"
 
 #include "Genotype.h"
 #include "DataLikelihood.h"
diff -Nru freebayes-1.3.5/src/Genotype.cpp freebayes-1.3.5b/src/Genotype.cpp
--- freebayes-1.3.5/src/Genotype.cpp	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/Genotype.cpp	2021-05-13 16:26:23.000000000 +0100
@@ -1,6 +1,6 @@
 #include "Genotype.h"
-#include "multichoose.h"
-#include "multipermute.h"
+#include "multichoose/multichoose.h"
+#include "multichoose/multipermute.h"
 
 
 vector<Allele*> Genotype::uniqueAlleles(void) {
diff -Nru freebayes-1.3.5/src/Parameters.h freebayes-1.3.5b/src/Parameters.h
--- freebayes-1.3.5/src/Parameters.h	2021-02-10 09:21:07.000000000 +0000
+++ freebayes-1.3.5b/src/Parameters.h	2021-05-13 16:26:24.000000000 +0100
@@ -9,7 +9,7 @@
 #include <map>
 #include <getopt.h>
 #include <stdlib.h>
-#include "split.h"
+#include "fastahack/split.h"
 #include "version_git.h"
 #include "Utility.h"
 
